en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -0,0 +1,116 @@
+
+4. First, we will create two defined sequences called `create.xml` and  `retrieve.xml` to create an Account and retrieve data. You can go to the source view of the XML configuration file of the API and copy the following configuration. Replace the `clientSecret`, `clientId`, `accessToken`, `refreshToken` with obtained values in step 3. 
+
+create.xml
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="create" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <property expression="json-eval($.sObject)" name="sObject" scope="default" type="STRING"/>
+    <property expression="json-eval($.fieldAndValue)" name="fieldAndValue" scope="default" type="STRING"/>
+    <salesforcerest.init>
+        <accessToken></accessToken>
+        <apiVersion>v44.0</apiVersion>
+        <hostName>https://login.salesforce.com</hostName>
+        <refreshToken></refreshToken>
+        <clientSecret></clientSecret>
+        <clientId></clientId>
+        <apiUrl>https://ap16.salesforce.com</apiUrl>
+        <registryPath>connectors/SalesforceRest</registryPath>
+    </salesforcerest.init>
+    <salesforcerest.create>
+        <sObjectName>{$ctx:sObject}</sObjectName>
+        <fieldAndValue>{$ctx:fieldAndValue}</fieldAndValue>
+    </salesforcerest.create>
+</sequence>
+```
+
+retrieve.xml
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="retrieve" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <property expression="json-eval($.sObject)" name="sObject" scope="default" type="STRING"/>
+    <property expression="json-eval($.fieldAndValue)" name="fieldAndValue" scope="default" type="STRING"/>
+    <salesforcerest.init>
+        <accessToken></accessToken>
+        <apiVersion>v44.0</apiVersion>
+        <hostName>https://login.salesforce.com</hostName>
+        <refreshToken></refreshToken>
+        <clientSecret></clientSecret>
+        <clientId></clientId>
+        <apiUrl>https://ap16.salesforce.com</apiUrl>
+        <registryPath>connectors/SalesforceRest</registryPath>
+    </salesforcerest.init>
+    <salesforcerest.query>
+        <queryString>select id, name from Account</queryString>
+    </salesforcerest.query>
+</sequence>
+```
+
+5. Now create the API **salesforcerest**. You can go to the source view of the XML configuration file of the API and copy the following configuration. 
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/salesforcerest" name="salesforcerest" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST">
+        <inSequence>
+            <sequence key="create"/>
+            <sequence key="retrieve"/>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+</api>
+```
+8. Follow these steps to export the artifacts. 
```

Comment:
Update so that this is a different section. See File Connector Example.

Reply:
Done